include/clang/Driver/Options.td

```diff
@@ -1005,6 +1005,8 @@ def flto_jobs_EQ : Joined<["-"], "flto-jobs=">,
 def fthinlto_index_EQ : Joined<["-"], "fthinlto-index=">,
   Flags<[CC1Option]>, Group<f_Group>,
   HelpText<"Perform ThinLTO importing using provided function summary index">;
+def fstruct_field_cache_analysis : Flag<["-"], "fstruct-field-cache-analysis">, Group<f_Group>,
```

Comment:
Your pass is going to print out a report and not do any transforms IIUC. Look at Diag_group in Options.td file. In fact you should be just able to use the -Rpass option and extend it to dump your pass's reports